modules.d/45url-lib/module-setup.sh

```diff
@@ -32,9 +32,9 @@ install() {
         for _lib in "$dracutsysrootdir$_dir"/libcurl.so.*; do
             [[ -e $_lib ]] || continue
             if ! [[ $_nssckbi ]]; then
-                read -r -d '' _nssckbi < <(grep -F --binary-files=text -z libnssckbi "$_lib")
+                read -r -d '' _nssckbi <<< "(grep -F --binary-files=text -z libnssckbi $_lib)"
             fi
-            read -r -d '' _crt < <(grep -F --binary-files=text -z .crt "$_lib")
+            read -r -d '' _crt <<< "(grep -F --binary-files=text -z .crt $_lib)"
```

Comment:
```suggestion
            read -r -d '' _crt <<< "$(grep -F --binary-files=text -z .crt "$_lib")"
```

Reply:
Adding your suggested change triggers...
```
/45url-lib/module-setup.sh: line 37: warning: command substitution: ignored null byte in input
/usr/lib/dracut/modules.d/45url-lib/module-setup.sh: line 37: warning: command substitution: ignored null byte in input
/usr/lib/dracut/modules.d/45url-lib/module-setup.sh: line 37: warning: command substitution: ignored null byte in input
/usr/lib/dracut/modules.d/45url-lib/module-setup.sh: line 37: warning: command substitution: ignored null byte in input
```

Anyways it's probably best we either merge this as is since it's working and you then do a follow if necessary or we nack this since my brain is not fully working...

Reply:
NACK NACK NACK, your patch is incorrect.

Reply:
The `ignored null byte in input` is expected and could be worked around with `tr`... which is also done in our code base before... but this all papers over the bad chroot setup.

Reply:
papering over it with tr did not work I tried...